src/niveristand/_translation/utils.py

```diff
@@ -26,54 +27,58 @@ def get_value_from_node(node, resources):
                     datavalue = [0.0]
                 else:
                     datavalue = 0.0
-            elif type(node.args[0]) in (ast.Num, ast.UnaryOp):
+            elif type(node.args[0]) is ast.UnaryOp or is_node_ast_num(node.args[0]):
                 datavalue = get_element_value(node.args[0])
             elif type(node.args[0]) is ast.Name:
                 if node.args[0].id in symbols._symbols:
                     datavalue = symbols._symbols[node.args[0].id]
                 else:
                     raise TranslateError(_errormessages.init_var_invalid_type)
-            elif type(node.args[0]) is ast.NameConstant:
-                if node.args[0].value is True or node.args[0].value is False:
-                    datavalue = node.args[0].value
+            # In Python 3.8, NameConstant is Constant
```

Comment:
Extra comment.

Reply:
![image](https://user-images.githubusercontent.com/44372517/94543425-bad49580-0252-11eb-8692-d2a0757d4f0d.png)

In tests/test_multitask.py, it says that redefinition is exactly what is tested, so I guess I should not make any changes in that file.